src/WebJobs.Script/Binding/TableBinding.cs

```diff
@@ -184,11 +198,31 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
                 }
                 else
                 {
-                    // binding to entire table (query multiple table entities)
-                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName, boundPartitionKey, boundRowKey), additionalAttributes);
+                    // binding to multiple table entities
+                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName), additionalAttributes);
```

Comment:
Is there a bug in the WebJobs SDK when using PartitionKey? This seems to be impacting C# as well, since it relies on the attribute only. Do we need to fix that in the SDK to avoid having to perform a query here?